pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/design/CyclomaticComplexityRule.java

```diff
@@ -128,7 +126,7 @@ public Object visit(ASTAnyTypeDeclaration node, Object data) {
             int classWmc = (int) MetricsUtil.computeMetric(JavaClassMetricKey.WMC, node, cycloOptions);
 
             if (classWmc >= classReportLevel) {
-                int classHighest = (int) JavaMetrics.get(JavaOperationMetricKey.CYCLO, node, cycloOptions, ResultOption.HIGHEST);
+                int classHighest = (int) MetricsUtil.computeStatistics(JavaOperationMetricKey.CYCLO, node.getOperations(), cycloOptions).getMax();
```

Comment:
Seems like, we also need to update the doc for PMD 7: https://github.com/pmd/pmd/blob/pmd/7.0.x/docs/pages/pmd/userdocs/extending/metrics_howto.md
The examples need to be updated to use MetricsUtil instead of JavaMetrics

Reply:
Will do

Reply:
Cool, thanks!